quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapperGrouping.java

```diff
@@ -0,0 +1,13 @@
+package org.keycloak.quarkus.runtime.configuration.mappers;
+
+import org.keycloak.quarkus.runtime.cli.Picocli;
+
+public interface PropertyMapperGrouping {
+
+    PropertyMapper<?>[] getPropertyMappers();
```

Comment:
```suggestion
    List<PropertyMapper<?>> getPropertyMappers();
```

Would it be possible to use some collection? List? 

Please add some javadoc.

Reply:
> Would it be possible to use some collection? List?

Doesn't seem like it will add much value to make that change. What are your thoughts?

> Please add some javadoc.

Sure, I can add that.

Reply:
@shawkins Once we're creating such an interface, I'd prefer to use some collection instead of plain arrays. 

There should not be any performance implications by doing that, right?

I'd rather see `List.of(...)` instead of `new PropertyMapper[] {...}`.